os-list.csv

```diff
@@ -1,5 +1,6 @@
 centos,7
 centos,6
+debian,buster
 debian,jessie
 debian,stretch
 fedora,28
```

Comment:
Seems like removing Fedora also makes sense, since we don't release it anymore, IIRC…